### PR TITLE
added contributing.md

### PR DESCRIPTION
**Pull request by onemriganka**
fixes: #74 

Hello, I added a simple contributing. md file. which i think helpful for all the beginners to contribute to this project

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+- Give the PR a descriptive title.
+- Provide a clear and detailed description of your changes.
+- Reference related issues, if any.
+- Be prepared to respond to feedback and make necessary adjustments.
+
+### Issues
+
+If you find a bug, have a suggestion, or want to work on an issue, please check the project's issue tracker. If the issue doesn't exist yet, feel free to open a new one to discuss the problem or feature request.
+
+## Community and Communication
```
Comment (rohan472000) [inline CONTRIBUTING.md:88]: I don't have any discord, better write that reach out through my mail.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+### Fork the Repository
+
+Fork our repository by clicking the "Fork" button on the top right of the repository page. This will create a copy of the project in your GitHub account.
+
+### Clone the Repository
+
+After forking the repository, clone it to your local machine using the following command, replacing `[your-username]` with your GitHub username:
+
+```bash
+git clone https://github.com/[your-username]/[repository-name].git
```
Comment (rohan472000) [inline CONTRIBUTING.md:42]:
```suggestion
git clone https://github.com/rohan472000/rohan472000.git
```

**Review by rohan472000 — CHANGES_REQUESTED** [review]

**Comment by onemriganka** [comment]
I have updated the contributing.md as you told me. is it okay now? or does anything have to be done again? I will correct if I made any mistake

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+## Getting Started
+
+### Prerequisites
+
+Before you start contributing, please make sure you have the following:
+
+- [Git](https://git-scm.com/) installed
+- A text editor or integrated development environment (IDE) of your choice
+- Knowledge of the programming language(s)  
+- Knowledge of python and ci CD with GitHub actions
```
Comment (rohan472000) [inline CONTRIBUTING.md:31]:
```suggestion
- Knowledge of python and CI/CD with GitHub actions
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+### Fork the Repository
+
+Fork our repository by clicking the "Fork" button on the top right of the repository page. This will create a copy of the project in your GitHub account.
+
+### Clone the Repository
+
+After forking the repository, clone it to your local machine using the following command, replacing `[your-username]` with your GitHub username:
```
Comment (rohan472000) [inline CONTRIBUTING.md:39]:
```suggestion
After forking the repository, clone it to your local machine using the following command:
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (rohan472000) [reply CONTRIBUTING.md:88]: remove this community and give my mail for communication (anand00rohan@gmail.com)

**Review by rohan472000 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**